### PR TITLE
App: Boards: Set CONFIG_COMP_DRC=y for LNL platform

### PR DESCRIPTION
**Pull request by singalsu**
The DRC component was not included to build. This change avoids error and DSP panic in topology load.

### CODE REVIEW

File: app/boards/intel_adsp_ace20_lnl.conf [hunk 1]
```diff
@@ -2,8 +2,8 @@ CONFIG_LUNARLAKE=y
 CONFIG_IPC_MAJOR_4=y
 
 CONFIG_COMP_SRC=y
-
 CONFIG_COMP_SRC_IPC4_FULL_MATRIX=y
+CONFIG_COMP_DRC=y
```
Comment (plbossart) [inline app/boards/intel_adsp_ace20_lnl.conf:6]: Alternatively should this be a library @singalsu ?

I thought this was the direction of travel for all processing components?

Comment (kv2019i) [reply app/boards/intel_adsp_ace20_lnl.conf:6]: @plbossart we are not there yet @lyakh correct me if I'm wrong

**Review by kv2019i — APPROVED** [review]

File: app/boards/intel_adsp_ace20_lnl.conf (hunk 1 above)
Comment (plbossart) [reply app/boards/intel_adsp_ace20_lnl.conf:6]: Didn't @lgirdwood say that we would only test volume as separate components though? Maybe I am confused on timelines, but do we agree that at some point all those processing should come as libraries?

**Review by plbossart — APPROVED** [review]
no objection to merge to unlock daily tests, but I'd like clarity on the directions for processing components (built-in with Kconfig or libraries) and by when. Thanks!

File: app/boards/intel_adsp_ace20_lnl.conf (hunk 1 above)
Comment (lyakh) [reply app/boards/intel_adsp_ace20_lnl.conf:6]: @plbossart you mean as loadable libraries / modules? @kv2019i is right - **in principle** it's all there and working, there are just a couple of "temporary minor issues," e.g. ATM it's only working with open-source toolchains like the Zephyr one, not with the Cadence Xplorer based one...

Comment (plbossart) [reply app/boards/intel_adsp_ace20_lnl.conf:6]: Humm, then the stated goal of testing volume components only is not realistic until all dependencies are handled. Clearly the use of the Cadence toolchain is a key requirement, we don't release anything based on open-source toolchains.

**Pull request merged**